src/vs/base/browser/ui/splitview/splitview.ts

```diff
@@ -52,6 +53,10 @@ export interface IView {
 	setVisible?(visible: boolean): void;
 }
 
+export interface ISerializableView extends IView {
```

Comment:
This is introducing the concept of `serialization` to the SplitView.
I think the `SplitView` should have no notion of this concept, but instead the clients of the `SplitView`

Reply:
@isidorn This is what @sbatten and I discussed: the splitview needs to be serializable to be able to hydrate in a performant manner.